ci-operator/config/jianzhangbjz/learn-operator/jianzhangbjz-learn-operator-master.yaml

```diff
@@ -0,0 +1,76 @@
+base_images:
+  base:
+    name: "4.7"
+    namespace: ocp
+    tag: base
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.14
+images:
+- dockerfile_path: build/Dockerfile.ci
+  from: base
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /tmp/learn-operator
+      - destination_dir: .
+        source_path: /tmp/bin
+  to: learn-operator
+operator:
+  bundles:
+  - context_dir: manifests
+    dockerfile_path: bundle.Dockerfile
+  substitutions:
+  - pullspec: quay.io/olmqe/learn-operator:v1
+    with: learn-operator
+promotion:
+  name: learn
+  namespace: ocp
+releases:
+  latest:
+    release:
+      channel: stable
+      version: "4.6"
+  previous:
+    candidate:
+      architecture: amd64
+      product: ocp
+      relative: 1
+      stream: nightly
+      version: "4.7"
```

Comment:
Why did you add this? You are apparently not using it. Additionally, `previous` being a 4.7 nightly to a `latest` being `4.6` stable makes no sense.

Reply:
Originally, I just hope to use a 4.7 `nightly` payload as a test cluster. Remove it.

Reply:
That's definitely possible. The version of the cluster that will be installed is determined by the release called `latest` - and you configured stable 4.6 under that name,

Reply:
Thanks! I think I totally understand the `initial`, `candidate`, `prerelease` functions by reading the https://github.com/openshift/ci-tools/tree/master/pkg/release code. In the beginning, I thought the `candidate` is only for the `previous` field based on the https://docs.ci.openshift.org/docs/architecture/ci-operator/#testing-with-an-existing-openshift-release. That example is really confusing for the beginner. Do you know where can I submit the PR for the example and doc updates? Thanks! I want to add more explanation for them so that the followers more clear.

Reply:
For docs, follow the big "GitHub" link at the top of the page, or the Edit this page / Create documentation issue links in the menu on the right.

Reply:
Got it. Thanks very much!